crates/bevy_render/Cargo.toml

```diff
@@ -31,9 +31,8 @@ image = { version = "0.23.12", default-features = false }
 # misc
 serde = { version = "1", features = ["derive"] }
 bitflags = "1.2.1"
-smallvec = "1.4.2"
-# TODO: replace once_cell with std equivalent if/when this lands: https://github.com/rust-lang/rfcs/pull/2788
-once_cell = "1.4.1"
+smallvec = { version = "1.6", feautres = ["union", "const_generics"] }
```

Comment:
typo found: `feautres` => `features`.

cargo is complaining about this "unused manifest key."

Reply:
could be fixed in https://github.com/bevyengine/bevy/pull/2111